Sources/InlineSnapshotTesting/AssertInlineSnapshot.swift

```diff
@@ -114,7 +116,7 @@ import Foundation
         )
         return
       }
-      guard let difference = snapshotting.diffing.diff(actual, expected)?.0
+      guard let difference = snapshotting.diffing.diff(expected, actual)?.0
```

Comment:
This diff felt backwards in the error message, so this should improve the readability.